source-code/app/src/main/java/org/buildmlearn/toolkit/activity/FirstRunActivity.java

```diff
@@ -44,11 +45,37 @@ protected void onCreate(Bundle savedInstanceState) {
         }
         setContentView(R.layout.activity_first_run);
 
-
+        next = (Button) findViewById(R.id.button_next);
         findViewById(R.id.focus_thief).clearFocus();
-        Animation animBounceinup=AnimationUtils.loadAnimation(getBaseContext(),R.anim.bounceinup);
+        Animation anim_bounceinup=AnimationUtils.loadAnimation(getBaseContext(),R.anim.bounceinup);
```

Comment:
Ok. On reviewing the project, it came to my notice that at many places snake cases have been used. At nowhere I used it. What I propose is as sson as I get green flag from Travis and cadacy bots,  I'll create another issue clearing it all over. If that is okay?